packages/core/test/percy.test.js

```diff
@@ -1301,6 +1301,40 @@ describe('Percy', () => {
       });
     });
 
+    describe('check for throttle logs endpoint from CLI', () => {
+      let maxSuggestionCalls = 10;
+      it('should increment suggestionsCallCounter and log a warning when the rate limit is reached', async () => {
+        spyOn(percy.client, 'getErrorAnalysis').and.returnValue([{
+          suggestion: 'some suggestion',
+          failure_reason: 'some failure reason'
+        }]);
+        percy.loglevel('debug');
+
+        for (let i = 0; i < maxSuggestionCalls; i++) {
+          await expectAsync(percy.suggestionsForFix('some_error')).toBeResolved();
+        }
+
+        await expectAsync(percy.suggestionsForFix('some_error')).toBeResolved();
+        expect(logger.stderr).toEqual(jasmine.arrayContaining([
+          '[percy:core] Rate limit exceeded for Maximum allowed suggestions per build.'
+        ]));
+      });
+
+      it('should not call getErrorAnalysis after exceeding the rate limit', async () => {
```

Comment:
can you update this case for checking multiple logs are not getting printed if we cross threshold